third-party/CMakeLists.txt

```diff
@@ -70,20 +70,17 @@ set(HYBRIDSQL_ASSERTS_HOME https://github.com/4paradigm/hybridsql-asserts)
 
 if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
```

Comment:
check `/deps/usr` to see whether the thirdparty deps are already downloaded.

Reply:
third-party is a bit general module to download/setup thirdparty, it do not have any assume of where that thirdparty located,  and should not contains hard link with other presets. 
instead, `/deps/usr` is checked in top `Makefile`.